### PR TITLE
added ui for home page

### PR DESCRIPTION
**Pull request by JantuDeb**
# Home Page
- Added UI for home page
- Category component.

image
image

### CODE REVIEW

File: src/pages/home/Home.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.videos {
+  display: grid;
+  grid-gap: 1rem;
+  /* grid-template-columns: repeat(1, 1fr); */
```
Comment (Manju-sree) [inline src/pages/home/Home.css:4]: good to see you have used grid to display video cards ,please remove commented lines

**Review by Manju-sree — APPROVED** [review]
I like that you have used CL css in this project and maintained proper folder structure and written clean readable code, and also i like the way you have breaking down components into many small components like horizontal and vertical card. it's good to merge.

File: src/components/shared/VideoCard.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+.video-title { 
+  font-size: 1rem;
+  overflow: hidden;
+  display: block;
+  -webkit-line-clamp: 2;
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+  text-overflow: ellipsis;
+  white-space: normal;
```
Comment (rkapoor10) [inline src/components/shared/VideoCard.css:24]: liked this use of css property

File: src/components/category/Category.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  border-radius: 10px;
+  width: 5rem;
+}
+
+
+.category-container li {
+  background-color: var(--fu-gray-800);
+  white-space: nowrap;
+  word-break: unset;
+  padding: 0.2rem 1rem;
```
Comment (rkapoor10) [inline src/components/category/Category.css:25]: learned something new..great work!

**Review by rkapoor10 — APPROVED** [review]
The code looks good to merge. Learned new things.

**Pull request merged**